app/helpers/task_helper.rb

```diff
@@ -0,0 +1,2 @@
+module TaskHelper
+end
```

Comment:
You can remove these empty modules. This is usually why we tend not to use the generators. They generate a bunch of other stuff that's often not needed. You can still use them if it helps though.